abc_spec/components/schemas/HostedPayments/HostedPaymentsRequest.yaml

```diff
@@ -180,3 +180,13 @@ properties:
         description: Determines whether to attempt a 3D Secure payment as non-3D Secure should the card issuer not be enrolled
         default: false
         example: false
+  capture:
+    type: boolean
+    description: Whether to capture the payment (if applicable)
+    example: true
+  capture_on:
+    description: |
+      A timestamp (ISO 8601 code) that determines when the payment should be captured.
+      Providing this field will automatically set `capture` to true
```

Comment:
Should we include something in the descriptions to say if either of these properties are provided it will override the merchant config?

And is the `Providing this field will automatically set `capture` to true` part relevant here?

Reply:
```suggestion
  capture:
    type: boolean
    description: |
      Whether to capture the payment (if applicable)
      Providing this field will override the `capture` field in the merchant configuration
    example: true
  capture_on:
    description: |
      A timestamp (ISO 8601 code) that determines when the payment should be captured.
      Providing this field will override the `capture_on` field in the merchant configuration
```

Reply:
I think there's potential for confusion here as the merchant config could be abstracted away from how a merchant developer interacts with this api since that's mostly handled by the solutions engineering team. Since we already have properties that follow this same logic (i.e. can be used to override the merchant config) like the `3ds ` object, let's just follow the wording we already have for those, which is to not mention merchant config.

I think we can keep the phrasing about automatically setting `capture` to `true` if that is how this logic works and keep us aligned to how that is presented on the `/payments` endpoint (ignoring the `capture` default they have of course)
![image](https://user-images.githubusercontent.com/59569871/132229753-29c21fc2-971f-4d73-98c4-59f172577e20.png)

Reply:
Good points, no changes needed then